packages/documentation-site/patternfly-docs/content/design-guidelines/styles/typography/typography.md

```diff
@@ -3,177 +3,80 @@ id: Typography
 
-<TitleLevel
-  className="pf-v6-c-title pf-m-lg ws-title-level-heading"
-  asGrid
-  styleProps={styleProps.third} />
-<div class="pf-v6-u-mt-lg">
-</div>
+### Font sizing: rem vs pixel
 
-### Fourth level heading (md, H4)
+Font size tokens use rems, rather than pixels. Rems are relative units that adjust font size based on a webpage's HTML document root element size. For example, if the root size is 10px, a rem size of 1.5 would be 15px.
 
-<TitleLevel
-  className="pf-v6-c-title pf-m-lg ws-title-level-heading"
-  asGrid
-  styleProps = {styleProps.fourth} />
-<div class="pf-v6-u-mt-lg">
-</div>
+PatternFly's default root element size is 16px. If you change this default size, note that the following tables will no longer show accurate pixel measurements (though the rem values will stay the same). 
 
-### Default body text
+## Headings 
 
-**Note:** This is the default text style for paragraphs, lists, tables, etc. Some components instead use Red Hat Text at 700 font weight, which is Red Hat Text Medium (such as alerts and navigation).
+All headings use Red Hat Display bold.
 
-<TitleLevel
-  className="pf-v6-c-title pf-m-lg ws-title-level-heading"
-  asGrid
-  styleProps = {styleProps.body} />
-<div class="pf-v6-u-mt-lg">
-</div>
+| Example | Tokens | Size | Line height | Usage | 
+| --- | --- | --- | --- | --- |
+| <Title headingLevel="h5" size='2xl'> Aa </Title> | --pf-t--global--font--size--heading--h1 | 1.375rem (22px) | 1.3 | Super hero headings <br /> H1 <br /> Page titles |
+| <Title headingLevel="h5" size='xl'> Aa </Title>  | --pf-t--global--font--size--heading--h2 | 1.25rem (20px) | 1.3 | Hero headings <br /> H2 |
+| <Title headingLevel="h5" size='lg'> Aa </Title>  | --pf-t--global--font--size--heading--h3 | 1.125rem (18px) | 1.3 | H3 |
+| <Title headingLevel="h5" size='md'> Aa </Title>  | --pf-t--global--font--size--heading--h4 <br /> --pf-t--global--font--size--heading--h5 <br /> --pf-t--global--font--size--heading--h6 |1rem (16px) | 1.3 | H4 <br /> H5 <br /> H6 |
 
-### Small text
+### Customizing heading levels
+The [title component](/components/title) allows you to customize the visual hierarchy of text on your page, while keeping the semantic hierarchy consistent with expectations for accessibility. If you use the title component, you can change the text size of different heading levels to customize beyond the default behavior.
 
-<TitleLevel
-  className="pf-v6-c-title pf-m-lg ws-title-level-heading"
-  asGrid
-  styleProps = {styleProps.small} />
-<div class="pf-v6-u-mt-lg">
-</div>
+For example, you may decide that the default size of 16px for secondary headings is too small and you want to increase the size from 16px (md) to 18px (lg). Rather than make your secondary headings H3’s, you should use the title component to keep them as H4 headings, but change the associated text size from `md` to `lg`. For example: 
 
-### Tiny text
+<Title headingLevel="h4" size='md'> This is an "md" H4. </Title>
+<br />
```

Comment:
These extra `<br/>` tags should no longer be necessary 🤞🏻 and it will reduce the white space a bit to remove them.